nixos/modules/services/x11/desktop-managers/pantheon.nix

```diff
@@ -168,12 +168,35 @@ in
       services.udev.packages = [
         pkgs.gnome.gnome-settings-daemon338
       ];
-      systemd.packages = [
-        pkgs.gnome.gnome-settings-daemon338
+      systemd.packages = with pkgs; [
+        gnome.gnome-settings-daemon338
+        gnome.gnome-session
+        pantheon.gala
+        pantheon.elementary-session-settings
       ];
       programs.dconf.enable = true;
       networking.networkmanager.enable = mkDefault true;
 
+      systemd.user.targets."gnome-session-initialized".wants = [
+        "org.gnome.SettingsDaemon.A11ySettings.target"
+        "org.gnome.SettingsDaemon.Color.target"
+        "org.gnome.SettingsDaemon.Datetime.target"
+        "org.gnome.SettingsDaemon.Housekeeping.target"
+        "org.gnome.SettingsDaemon.Keyboard.target"
+        "org.gnome.SettingsDaemon.MediaKeys.target"
+        "org.gnome.SettingsDaemon.Power.target"
+        "org.gnome.SettingsDaemon.PrintNotifications.target"
+        "org.gnome.SettingsDaemon.Rfkill.target"
+        "org.gnome.SettingsDaemon.ScreensaverProxy.target"
+        "org.gnome.SettingsDaemon.Sharing.target"
+        "org.gnome.SettingsDaemon.Smartcard.target"
+        "org.gnome.SettingsDaemon.Sound.target"
+        "org.gnome.SettingsDaemon.UsbProtection.target"
+        "org.gnome.SettingsDaemon.Wacom.target"
+        "org.gnome.SettingsDaemon.Wwan.target"
+        "org.gnome.SettingsDaemon.XSettings.target"
```

Comment:
Ooops `org.gnome.SettingsDaemon.Wwan.target` is not in `/etc/systemd/user/gnome-session@gnome.target.d/gnome.session.conf`...

And seems that `systemd.packages` isn't picking `.wants` directories...

Reply:
Maybe `nixos/modules/services/desktops/gnome/gnome-settings-daemon.nix` needs cleanup, I opened #153840, I think I also need a `services.gnome.gnome-settings-daemon.package` option in the future, maybe I will do that when this PR is ready.

Reply:
See also https://github.com/NixOS/nixpkgs/commit/68ab37aa4491c8f870483c9eff888dc5fc88d2c8, not sure what the reasoning was any more.

Reply:
Ooops, then I guess https://github.com/NixOS/nixpkgs/pull/153840 itself is a reason...